src/contacts/MultiContactViewer.js

```diff
@@ -28,11 +28,14 @@ export class MultiContactViewer {
 				m(".fill-absolute.mt-xs.plr-l",
 					(contactView._contactList && contactView._contactList.list.getSelectedEntities().length > 0) ? [
 							// Add spacing so buttons for contacts also align with the regular client view's buttons
-							m(".header.pt-ml",
-								m(".contact-actions.flex-space-between.flex-wrap", [
-									m(".flex.items-center", this._getContactSelectionMessage(contactView)),
-									m(actionBar)
-								])
+							m(".header.pt-ml.flex-space-between",
+								m(".left.flex-grow-shrink-150", [
```

Comment:
I don't see why it should be `.flex-grow-shrink-150`, it can be just `.flex-grow`

Reply:
Got it!